src/ui/components/DataTable/CustomizeColumns.tsx

```diff
@@ -1,134 +1,151 @@
+			addColumns: string[] = difference(enabledColumns, currentColumns);
+
+		// Apply removals and adding to currentColumns while trying to preserve the order of currentColumns
+		const nextColumns: string[] = currentColumns.filter(
+			c => !removeColumns.includes(c),
+		);
+		nextColumns.push(...addColumns);
+
+		await toWorker("main", "updateColumns", {
+			columns: nextColumns,
+			key: config.tableName,
+		});
+		onHide();
+		onSave();
+	};
+
+	const colsGrouped: { [key: string]: ColConfig[] } = groupBy(
+		columns,
+		c => c.cat,
+	);
 
 	return (
-		<Modal animation={false} centered show={show} onHide={onHide}>
-			<Modal.Header closeButton>Customize Columns</Modal.Header>
+		<Modal
+			animation={false}
+			size={"lg"}
+			show={show}
+			onEscapeKeyDown={exit}
+			centered
+		>
+			<Modal.Header onHide={exit} closeButton>
+				Customize Columns
+			</Modal.Header>
 			<Modal.Body>
-				<p>
-					Click and drag to reorder columns, or use the checkboxes to show/hide
-					columns.
-				</p>
-				{hasSuperCols ? (
-					<p className="text-danger">
-						This table has two header rows. That means you can enable/disable
-						columns, but not reorder them.
-					</p>
-				) : null}
-				<Container
-					helperClass="sort-inside-modal"
-					isDragged={isDragged}
-					onSortStart={() => {
-						setIsDragged(true);
-					}}
-					onSortEnd={args => {
-						setIsDragged(false);
-						if (!hasSuperCols) {
-							onSortEnd(args);
-						}
-					}}
-				>
-					{colOrder.map(({ colIndex, hidden }, i) => {
-						const col = cols[colIndex];
-						return (
-							<Item
-								key={colIndex}
-								index={i}
-								onToggleHidden={onToggleHidden(i)}
-								hidden={hidden}
-								col={col}
-							/>
-						);
-					})}
-				</Container>
+				<ul className="list-group list-group-flush">
+					{Object.entries(colsGrouped).map(([group, cols]) => (
+						<li key={group} className="list-group-item">
+							<strong>{group}</strong>
+							<div className="row">
+								{cols.map((col, i) => (
+									<div key={i} className="col-lg-4">
+										<div className="form-check d-flex">
+											<input
+												id={`show-column-${col.key}`}
+												className="form-check-input cursor-pointer"
+												type="checkbox"
+												checked={!col.hidden}
+												onChange={onChange(col.key)}
+											/>
+											<label
+												className="form-check-label cursor-pointer user-select-none flex-grow-1"
+												htmlFor={`show-column-${col.key}`}
+												title={col.desc}
+											>
+												{col.title}
+												<small className="d-lg-none ml-2">{col.desc}</small>
+											</label>
+										</div>
+									</div>
+								))}
+							</div>
+						</li>
+					))}
+				</ul>
 			</Modal.Body>
-			<Modal.Footer>
-				<button className="btn btn-danger" onClick={onReset}>
+			<Modal.Footer className="d-flex justify-content-between">
```

Comment:
Nowhere else in the UI has left/center/right buttons.

As a user, I have no idea what the difference between "Reset" and "Restore Default" is.

Maybe make it look more like this from the "Customize Teams" screen when creating a new league:

![image](https://user-images.githubusercontent.com/828632/150885383-b9b06757-598a-4461-a327-70ca4987beef.png)

So "Reset" is renamed to "Cancel" and moved to the right with "Save". And "Restore Default" is moved to the left, maybe renamed "Reset to Default", class btn-danger, and ideally should keep this form open and with unsaved changes after pressing it.

Reply:
Reset - reverts the modal to what it looked like when you opened it (so basically undoes any changes that aren't saved)
Restore Default - removes the entire configuration record from `meta.tables` so it uses the fallback configuration (basically it looks the same as if you just started up bbgm for the first time)

Reply:
![image](https://user-images.githubusercontent.com/45333542/150999535-5d745f58-370c-4aa0-a999-4a3f0be15e6c.png)
added a help popover to try and explain this to the user